### DISCUSSION TITLE
What was the batch size in the Pre-trained wights training?

### DISCUSSION THREAD

**Discussion by assafzamir** [ultralytics/ultralytics#3896] [Docs]
What was the batch size in the Pre-trained weights training process?

**Comment by glenn-jocher** [comment]
@assafzamir the pre-trained weights for YOLOv8 were typically trained with a batch size that maximizes the utilization of the available GPU memory. This can vary depending on the specific hardware and setup used during training. For exact details, you can refer to the model's configuration file or the training logs, which often include such information. If you're looking to replicate or understand the training conditions for the pre-trained models, the configuration files are a good starting point. 📘
